test/integration/targets/lvg/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+- name: Install required packages (Linux)
+  package:
+    name: lvm2
+    state: present
+  when: ansible_system == 'Linux'
+
+- name: Test lvg module
+  block:
+    - name: Create file to use as a disk device
+      command: "dd if=/dev/zero of={{ ansible_user_dir }}/ansible_testing/img1 bs=1M count=10"
```

Comment:
I think there is a var that represents a temp directory, though from a quick look I can't see it.